pkg/apis/policy/v1beta1/clusterimagepolicy_validation.go

```diff
@@ -60,10 +61,10 @@ func (image *ImagePattern) Validate(ctx context.Context) *apis.FieldError {
 	var errs *apis.FieldError
 	if image.Glob == "" {
 		errs = errs.Also(apis.ErrMissingField("glob"))
+	} else {
```

Comment:
nit:  It looks like we could have a return in the if statement `if image.Glob == "" { return errs.Also(apis.ErrMissingField("glob"))` so we don't need the following else statement.

Reply:
Yeah, I went with this way so that we could collect more errors below the bit I made an `else`, but since this is just for globs, IDK that there'll be much else.

The new `glob.Compile` was just piling on when complaining about empty-string, which I thought was unnecessarily confusing.  I'll switch these to returns instead.

Reply:
Should be RFAL 🙏